README.beginner-developer

```diff
@@ -0,0 +1,72 @@
+    This is some collection to help recently joined coders to start with.
+    You can try the hard way. Then you just do not need to read any further.
+    Beleive me, at least for me, adds unnecessary frustration.
```

Comment:
Well, among four of us, please let me explain my frustration. Not professional behaviour, I admit, and must excuse me for it...

Today I have somehow found out my root problem to this. In mid-school, I had a subject (~course), technical drawing, screws and other mechanical things, wiring diagrams, etc. We regularly used to have homework. For some, it was always accepted. For others, usually including me, it was rejected with the worst note, and when asking, why, the response always sounding "the newest standards....". Never seen anybody ever having seen such standards, by the way ;-) . And each and every week other standards ;-) ... Well, it did not disturb us very much that time, saying "she is an idiot". Nor did I face any similar situation since then, almost 50 years now. However, with tlf, both my two contributions resulted in this... Well, I know I do not get bad notes any more. Well, I know, everybody helps me, and everybody is happy to see any useful contributions. Still the fact is, I did my best, Zoli has checked it, I only wanted to lean back and be proud and happy, and then still again I was asked to correct things that I could in no way know in advance.

Such a list like this could have helped me. Kindly ask you, to your best, do please support this for me and for all future mates.

Reply:
This is part of collaborative development.  I submitted some pull requests last month and there were some obvious things I missed that the others saw and their solutions were clearly better.  For me it is an opportunity to learn and get better especially since I am self-taught in programming with no formal education in the field whatsoever.  My studies were in electronics 30 years ago.  I still melt solder on occasion but I greatly enjoy working with a few software projects.

Reply:
Well, I also am ready to learn. However, unhappy when I suddenly get the feeling I should have known something nowhere documented... And that was my feeling at the end, no matter how nice the missing point communicated to me by nice fellows...